[Bug]: Could not resolve a module for a custom reporter => github-actions

### Description of the bug

Currently, when running the `tests-unit-js.yml` workflow, the following error is shown:

```
Error: Could not resolve a module for a custom reporter.
  Module name: github-actions
```

### Reproduction instructions

Push a commit to a repository using the `tests-unit-js.yml` action.

### Expected behavior

The GitHub action runs without issues.

### Environment info

_No response_

### Relevant log output

```shell
Run ./node_modules/.bin/jest --reporters=default --reporters=github-actions
  
Error: Could not resolve a module for a custom reporter.
  Module name: github-actions
    at /home/runner/work/project-name/project-name/node_modules/jest-config/build/normalize.js:558:15
    at Array.map (<anonymous>)
    at normalizeReporters (/home/runner/work/project-name/project-name/node_modules/jest-config/build/normalize.js:540:33)
    at normalize (/home/runner/work/project-name/project-name/node_modules/jest-config/build/normalize.js:633:5)
    at readConfig (/home/runner/work/project-name/project-name/node_modules/jest-config/build/index.js:224:68)
    at async readConfigs (/home/runner/work/project-name/project-name/node_modules/jest-config/build/index.js:406:26)
    at async runCLI (/home/runner/work/project-name/project-name/node_modules/@jest/core/build/cli/index.js:230:59)
    at async Object.run (/home/runner/work/project-name/project-name/node_modules/jest/node_modules/jest-cli/build/cli/index.js:163:37)
Error: Process completed with exit code 1.
```


### Additional context

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

--- a/docs/js.md
+++ b/docs/js.md
@@ -74,6 +74,8 @@ jobs:
 | `NODE_VERSION`        | 16                                                 | Node version with which the unit tests are to be executed |
 | `JEST_ARGS`           | `'--reporters=default --reporters=github-actions'` | Set of arguments passed to Jest                           |
 
+**Note**: The default `github-actions` reporter requires Jest 28 or higher.
+
 #### Secrets
 
 | Name                 | Description                                 |

